index.html

```diff
@@ -49,5 +49,167 @@ <h1 class="headline-title">Hey there. I'm Rachel</h1>
+                    </div>
+                    </article>
+                    <article>
+                        <div class="works-card">
+                            <div class="works-card-image">
+                                <img src="./img/Img Plaholder.svg" alt="">
+                            </div>
+                            <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                            <ul class="works-card-skills">
+                                <li>ruby on rails</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                                <li>html</li>
+                            </ul>
+                            <div><button type="button" class="project-btn">See Project</button></div>
+                        </div>
+                        </article>
+                        <article>
+                            <div class="works-card">
+                                <div class="works-card-image">
+                                    <img src="./img/Img Plaholder.svg" alt="">
+                                </div>
+                                <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                                <ul class="works-card-skills">
+                                    <li>ruby on rails</li>
+                                    <li>css</li>
+                                    <li>javascript</li>
+                                    <li>html</li>
+                                </ul>
+                                <div><button type="button" class="project-btn">See Project</button></div>
+                            </div>
+                            </article>
+                            <article>
+                                <div class="works-card">
+                                    <div class="works-card-image">
+                                        <img src="./img/Img Plaholder.svg" alt="">
+                                    </div>
+                                    <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                                    <ul class="works-card-skills">
+                                        <li>ruby on rails</li>
+                                        <li>css</li>
+                                        <li>javascript</li>
+                                        <li>html</li>
+                                    </ul>
+                                    <div><button type="button" class="project-btn">See Project</button></div>
+                                </div>
+                                </article>
+                            </div>
+                        </section>
+                        <!--ABOUT SECTION-->
+                        <section class="container-flex about-me">
+                                <div class="container">
+                                    <h2>About Me</h2>
```

Comment:
- [ ] as the previous reviewer requested kindly add the missing background image to follow Figma design

![image](https://user-images.githubusercontent.com/89282221/192094385-6070b799-dbc0-4682-a44a-7786d3a3b764.png)

Reply:
@ichala Hello reviewer! Unfortunately the figma templates website has been down since yesterday. I am unable to parse the image from my selected figma template due to this reason. How do i proceed please? 
Thanks, Rachel